docs/source/gbd2019_models/risk_effects/bmi/index.rst

```diff
@@ -0,0 +1,328 @@
+   * - High Body-Mass Index
+     - Breast cancer
+   * - High Body-Mass Index
+     - Uterine cancer
+   * - High Body-Mass Index
+     - Colon and rectum cancer
+   * - High Body-Mass Index
+     - Gallbladder and biliary tract cancer
+   * - High Body-Mass Index
+     - Pancreatic cancer
+   * - High Body-Mass Index
+     - Ovarian cancer
+   * - High Body-Mass Index
+     - Kidney cancer
+   * - High Body-Mass Index
+     - Thyroid cancer
+   * - High Body-Mass Index
+     - Multiple myeloma
+   * - High Body-Mass Index
+     - Ischemic heart disease
+   * - High Body-Mass Index
+     - Ischemic stroke
+   * - High Body-Mass Index
+     - Intracerebral hemorrhage
+   * - High Body-Mass Index
+     - Subarachnoid hemorrhage
+   * - High Body-Mass Index
+     - Hypertensive heart disease
+   * - High Body-Mass Index
+     - Atrial fibrillation and flutter
+   * - High Body-Mass Index
+     - Asthma
+   * - High Body-Mass Index
+     - Gallbladder and biliary diseases
+   * - High Body-Mass Index
+     - Alzheimer's disease and other dementias
+   * - High Body-Mass Index
+     - Chronic kidney disease due to hypertension
+   * - High Body-Mass Index
+     - Chronic kidney disease due to glomerulonephritis
+   * - High Body-Mass Index
+     - Chronic kidney disease due to other and unspecified causes
+   * - High Body-Mass Index
+     - Low back pain
+   * - High Body-Mass Index
+     - Gout
+   * - High Body-Mass Index
+     - Cataract
+   * - High Body-Mass Index
+     - Acute lymphoid leukemia
+   * - High Body-Mass Index
+     - Chronic lymphoid leukemia
+   * - High Body-Mass Index
+     - Acute myeloid leukemia
+   * - High Body-Mass Index
+     - Chronic myeloid leukemia
+   * - High Body-Mass Index
+     - Other leukemia
+   * - High Body-Mass Index
+     - Diabetes mellitus type 2
+   * - High Body-Mass Index
+     - Chronic kidney disease due to diabetes mellitus type 2
+   * - High Body-Mass Index
+     - Burkitt lymphoma
+   * - High Body-Mass Index
+     - Other non-Hodgkin lymphoma
+   * - High Body-Mass Index
+     - Osteoarthritis hip
+   * - High Body-Mass Index
+     - Osteoarthritis knee
+
+[GBD-2019-Capstone-Appendix-BMI2]_
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - Age group start
+     - 9
+     - [20, 24 years)
+   * - Age group end
+     - 235
+     - [95, 125 years)
+
+
+Risk Outcome Pair #1: Ischemic heart disease
+++++++++++++++++++++++++++++++++++++++++++++
+
+:ref:`See ischemic heart disease documentation <2019_cause_ihd>`
+
+The relative risks apply to the incidence rates of acute 
+myocardial infarction and stable angina. They should be 
+applied using the formula: 
+
+incidence(i) = incidence*(1-PAF\ :sub:`r108`\)*RR^{max((BMI_i - TMREL),0)/5} 
```

Comment:
I think this RR should be applied to arrow 1 (from S1 to AMI), arrow 3 (from P to AMI), and arrow 4 (from S2 to SA).  If I'm wrong about that, I won't be too surprised, but whatever we decide, we should be precise about which incidence we are referring to here.

Reply:
Thanks! Added